File: js210/js211_assessment/js210sp_easy5/sequence_count.js
"use strict";

function sequence(count, start) {
  let multiples = [];

  for (let index = 1; index <= count; index += 1) {
    let multiple = start * index;
    multiples.push(multiple);
  }

  return multiples;
}

console.log(sequence(5, 1));
console.log(sequence(4, -7));
console.log(sequence(3, 0));
console.log(sequence(0, 1000000));

File: rb109/Practice/Easy 9/sequence_count.rb
# sequence_count.rb

# create a method
# input: two separate integers
# first integer: a count
# second integer: first number of a sequence that my method creates
# output: an array containing the same number of elements as the count
# while the values of each element will be multiples of the starting number

def sequence(count, start_number)
  results = []
  1.upto(count) { |multiplier| results << start_number * multiplier }
  results
end

p sequence(5, 1) == [1, 2, 3, 4, 5]
p sequence(4, -7) == [-7, -14, -21, -28]
p sequence(3, 0) == [0, 0, 0]
p sequence(0, 1000000) == []
